Pods/Local Podspecs/StencilSwiftKit.podspec.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "StencilSwiftKit",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
All podspecs in `Pods/Local Podspecs` belong to pods in the file system.
Given that currently there are no local pods in `Podfile` those probably are leftovers from the development process.
I will remove them from git.